src/main/java/org/zaproxy/zest/core/v1/ZestExpressionClientElementExists.java

```diff
@@ -78,28 +78,11 @@ public boolean isTrue(ZestRuntime runtime) {
             return false;
         }
         String elem = runtime.replaceVariablesInString(this.getElement(), false);
-        By by = null;
-
-        if ("className".equalsIgnoreCase(type)) {
-            by = By.className(elem);
-        } else if ("cssSelector".equalsIgnoreCase(type)) {
-            by = By.cssSelector(elem);
-        } else if ("id".equalsIgnoreCase(type)) {
-            by = By.id(elem);
-        } else if ("linkText".equalsIgnoreCase(type)) {
-            by = By.linkText(elem);
-        } else if ("name".equalsIgnoreCase(type)) {
-            by = By.name(elem);
-        } else if ("partialLinkText".equalsIgnoreCase(type)) {
-            by = By.partialLinkText(elem);
-        } else if ("tagName".equalsIgnoreCase(type)) {
-            by = By.tagName(elem);
-        } else if ("xpath".equalsIgnoreCase(type)) {
-            by = By.xpath(elem);
-        } else {
+        By by = ZestScript.getBy(elem, type);
```

Comment:
Inline "elem" since it's only used once now?

Reply:
Yep - done.